Rakefile

```diff
@@ -1,8 +1,9 @@
-require 'bundler/setup'
```

Comment:
should always be there so we never run a different gem version / enforce bundler usage